ci/requirements_dev.txt

```diff
@@ -5,3 +5,4 @@ cython
 pytest>=3.1.0
```

Comment:
remove the no change files e.g

```
git checkout master ci/requirements-3.6_NUMPY_DEV.pip
```

Reply:
I added those files when I was attempting to install moto via pip in those environments; they don't exist in master. Do we want to get rid of them?

Reply:
actually these are empty, so ok